src/Core/Abstractions/IPreferencesStorageService.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Bit.Core.Abstractions
+{
+    public interface IPreferencesStorageService
```

Comment:
Can we give this interface a different name? It took me a moment to wrap my head around the need for it and I think it's because the inclusion of `Preferences` led me down a weird path. Perhaps something like `SynchronousStorageService`?

Reply:
What I like about the `Preferences` prefix is that the caller knows exactly where the data is stored, whereas with `Synchronous` it could be anywhere.
However, I see the point about causing confusion on when that should be used and your suggestion makes sense.
Either if we keep `Preferences` or we change it to `Synchronous` (or something else), I'd like to add a `summary` to the interface so that it explains it further. What do you think?
In addition, I'd like to refactor at some time the registration and interface of `IStorageService` so we don't have to keep relying on the registration key to determine which implementation is using, so we'd have additional specific interfaces for each implementation and the `IStorageService` as the base one.

Reply:
> What I like about the Preferences prefix is that the caller knows exactly where the data is stored, whereas with Synchronous it could be anywhere.

That's exactly what I'm thinking though - we already have a preference-specific storage service that implements `IStorageService` and you're adding synchronous functionality to it via another interface that could potentially be applied to the other non-preference storage services as well if the need arises.  In addition the authority that answers the question "where is this stored" is already implemented as `_preferenceStorageKeys` in `MobileStorageService`.  Does this make sense, or is my interpretation incorrect?